www/source/english.xml

```diff
@@ -610,7 +610,7 @@
         <message name="CompositeImageRequired">
           composite image required
         </message>
-        <message name="CurlyBrasesNestedTooDeeply">
+        <message name="CurlyBracesNestedTooDeeply">
```

Comment:
notable. these changes presumably break localizations -- each project has a different policy about localizations, it's a toss up between "please update" and "please don't touch"...